schema/types/isolator_linux_specific_test.go

```diff
@@ -94,3 +94,113 @@ func TestNewLinuxCapabilitiesRevokeSet(t *testing.T) {
 	}
 
 }
+
+func TestNewLinuxSeccompRemoveSet(t *testing.T) {
+	tests := []struct {
+		set   []string
+		errno string
+
+		expectedSet   []LinuxSeccompEntry
```

Comment:
you went in the wrong direction here - the rest of the file is pretty consistent with w/want :-(